### ISSUE TITLE
NPC definitions - repeated nodes/fleets vs single node/fleet with more ships

### ISSUE THREAD

**Issue by tehhowch** [endless-sky/endless-sky#2995]
@endless-sky While seeking places to cut duplicate fleets in #2993, I noticed a few instances of exactly duplicated NPC definitions, and it appears to have been an explicit decision to do so (https://github.com/endless-sky/endless-sky/commit/000d2ad3f6bf775e22eb5880f5aa1595781fc08a). 

That commit doesn't explain the behavioral changes that are caused by using separate NPC nodes rather than a single node (I can't think of any, as they all have the same `parent`, the player's flagship).
The same question applies to duplicated custom fleet specifications, as after instantiation the NPCs all have the same parent ship (the player's flagship) - it appears these could be trimmed simply by using more of the spawned ships.

e.g. What behavioral change is achieved with A vs B vs C?:
```
A
npc
    fleet
        variant
            "Super Ship 1"
    fleet
        variant
            "Super Ship 1"

B
npc
    fleet
        variant
            "Super Ship 1" 2

C
npc
    fleet
        variant
            "Super Ship 1"
npc
    fleet
        variant
            "Super Ship 1"
```

**Comment by endless-sky** [comment]
I think my goal was to break the fleets up into separate groups with their own flagships, but you're right that if they are set up to follow the player that won't matter.

**Comment by tehhowch** [comment]
If ai rework goes far enough to define "squadron" activity, we could use
separated definition blocks to define that behavior.

On Sep 2, 2017 5:53 PM, "Endless Sky" <notifications@github.com> wrote:

I think my goal was to break the fleets up into separate groups with their
own flagships, but you're right that if they are set up to follow the
player that won't matter.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/endless-sky/endless-sky/issues/2995#issuecomment-326773539>,
or mute the thread

**Comment by endless-sky** [comment]
Most of these would have been addressed in #2993, I think.
